test/benchmarks/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+time solc --via-ir --optimize --bin chains.sol
```

Comment:
Also, this requires having `solc` installed instead of using a fresh build as we generally do. I'll look into expanding the script a little bit to make it easier to use.